src/SDKs/DataFactory/Management.DataFactory/changelog.md

```diff
@@ -1,10 +1,13 @@
 # Changelog for the Azure Data Factory V2 .NET SDK
 
 ## Current version
+## Version 0.8.0-preview
```

Comment:
Do not add this. I will update the version later when we release. Leave your changes under Current version

Reply:
done